src/Symfony/Component/Dotenv/Command/DebugCommand.php

```diff
@@ -125,19 +128,36 @@ private function getEnvFiles(): array
         return $files;
     }
 
-    private function getFilePath(string $file): string
+    private function getFilePath(?string $dotenvDirectory, string $file): string
     {
-        return $this->projectDirectory.\DIRECTORY_SEPARATOR.$file;
+        return $this->projectDirectory.(null === $dotenvDirectory ? '' : \DIRECTORY_SEPARATOR.$dotenvDirectory).\DIRECTORY_SEPARATOR.$file;
     }
 
-    private function loadValues(string $file): array
+    private function loadValues(?string $dotenvDirectory, string $file): array
     {
-        $filePath = $this->getFilePath($file);
+        $filePath = $this->getFilePath($dotenvDirectory, $file);
 
         if (str_ends_with($filePath, '.php')) {
             return include $filePath;
         }
 
         return (new Dotenv())->parse(file_get_contents($filePath));
     }
+
+    private function getDotenvDirectory(): ?string
+    {
+        $projectDir = is_file($projectDir = $this->projectDirectory) ? basename($projectDir) : $projectDir;
+
+        $composerFile = $projectDir.'/composer.json';
+        if (is_file($composerFile)) {
+            $composerContent = json_decode(file_get_contents($composerFile), true);
+            $dotenvPath = $composerContent['extra']['runtime']['dotenv_path'] ?? null;
+
+            if (null !== $dotenvPath) {
+                return \dirname($dotenvPath);
+            }
+        }
+
+        return null;
```

Comment:
Return the default value.
```suggestion
        return '.env';
```